index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
+            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
+        </nav>
+        <section class="headline">
+            <header>
+                <h2>Hey there. I'm Saadat 
+                    <br>
+                    <span>I'm a software developer</span>
+                </h2> 
+            </header>
+            <p class="desc"> I can help you build a product, 
+                feature or website Look through
+                some of my work and experience! If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.</p>
+            <div class="social-icons">
+                <img src="img/twitter.svg" alt="Twitter">
+                <img src="img/LinkedinIcon.svg" alt="Linkedin">
+                <img src="img/medium.svg" alt="Medium">
+                <img src="img/github.svg" alt="Github">
+                <img src="img/angellist.svg" alt="Angellist">
+            </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social icons and use `li` tag to wrap each of the social icons instead of a `div` tag. This is because the social icons are unordered lists and therefore should be wrapped with a `ul` and `li` tag and also to improve the code structure.

 

- [ ] Also, each icon should be wrapped with an `a` tag which will link to your social media handles.

See my suggestion below:
```suggestion
            <ul class="social-icons">
              <li>
                <a href="">
                  <img src="img/twitter.svg" alt="Twitter">
                </a>
              </li>
              <li>
                <a href="">
                  <img src="img/LinkedinIcon.svg" alt="Linkedin">
                </a>
              </li>
              <li>
                <a href="">
                  <img src="img/medium.svg" alt="Medium">
                </a>
              </li>
              <li>
                <a href="">
                  <img src="img/github.svg" alt="Github">
                </a>
              </li>
              <li>
                <a href="">
                   <img src="img/angellist.svg" alt="Angellist">
                </a>
              </li>
            </ul>
```